pending/dancer_core_send_as.pod

```diff
@@ -0,0 +1,125 @@
+=encoding utf8
+
+=head2 send_as BEER => 'Christmas Cheer'
+
+One Frequently Asked Question that arose in 2015 from several
+keen Dancer2 users was 
+
+  "How to return JSON from a specific route when we have a template
+  engine configured?"
+
+The best advice back then was to "split" you app into two distinct
+applications and use C<Plack::Builder> to glue them back together.
+
+  # bin/app.psgi
+  use Plack::Builder;
+  use MyApp;
+  use MyApp::API;
+
+  builder {
+      mount '/api' => MyApp::API->to_app;
+      mount '/'    => MyApp->to_app;
+  };
+
+However if you have a small number of routes that required serialization
+(or a small number that needed a template engine), the effort involved
+was onerous.
+
+Other suboptimal solutions (including the use of eldritch horrors) were
+proposed in issues on github prior to a suggestion to use a combination of
+Dancer2's C<send_file> and C<to_json> using an in-memory file handle:
+
+  set template => 'TemplateToolkit';
+
+  get '/api/thing' => sub {
+      my $json = to_json { thing => { to => [ 'return' ] } };
+      content_type => 'application/json';
+      send_file \$json;     # sends json
+  };
+
+Which just works; but involves excessive boilerplate and one needs to take
+care with text encodings. That's not the Dancer way!
```

Comment:
An interesting perspective of this is that this works because `send_file` does not go through serializers.